osu.Game/Input/Handlers/ReplayInputHandler.cs

```diff
@@ -79,5 +80,38 @@ public ReplayStateChangeEvent(InputState state, IInput input, T[] releasedAction
                 PressedActions = pressedActions;
             }
         }
+
+        /// <summary>
+        /// An <see cref="IInput"/> that is triggered when a frame containing replay statistics arrives.
+        /// </summary>
+        public class ReplayStatisticsFrameInput : IInput
```

Comment:
Not super related but I find it kind of weird how this class is declared as inner here but only ever used in `FramedReplayInputHandler` but at least it's consistent I guess with the other inputs? I dunno, this class feels like a bit of a husk, `FramedReplayInputHandler` is its only direct inheritor and the only thing rulesets use. Maybe this particular class needs to be removed at some point in the future?

Reply:
Yeah I just copied the way the other input/event was done. Probably would agree with merging `ReplayInputHandler` and `FramedReplayInputHandler` as I don't think anything else makes sense anyway.